feat: Eraser size changes visible in Erasebypoint

nle244 opened the pull request:
Second attempt to address this issue. First try had conflicts. New to GitHub so hopefully this works.

Had to use a hacky way to change the size of the cursor since it cannot be updated dynamically as noted here:

https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.inkcanvas.erasershape?view=net-5.0#remarks

- Silently toggle EraseByPoint to enable dynamic resizing.
- Had to add two lines to EraserFunction to keep the circle shape otherwise the first time using EraseByPoint the cursor is square.

ab28dev commented:
While changing the size of eraser if I toggle to the minimum eraser size, the eraser econ is no more visible. Please correct this.

nle244 commented:
Hello, thanks for looking at this. When setting the eraser size the smallest eraser is mostly black due to the black border which will make it very hard to see on a dark background which I assume is the issue. When using the smallest sized eraser on a while background it will be visible. 

Additionally while reviewing my code I accidentally introduced another bug where the mode unexpectedly changes to EraseByPoint. It was a simple fix. Do I close this pull request and submit a new one? (Sorry, new to github)

ab28dev commented on LiveDraw/MainWindow.xaml.cs:
```diff
@@ -872,6 +872,7 @@ private void LineMode(bool l)
         {
             LineButton.IsActived = l;
             _lineMode = l;
+            //EraseByPoint_Flag = (int)erase_mode.NONE;
```
Don't comment this the flag is being used for eraser toggle.

ab28dev reviewed (changes requested):
Have you changed the minimum eraser size to 3px? I will test this if you have done so. 

Also how to reproduce the bug you found?

nle244 commented:
I changed the minimum pixel to 3 for you to test.

The bug I introduced was after using EraseByPoint once, resizing the cursor even in Line or Ink mode the cursor would switch back to EraseByPoint unexpectedly. But it's fixed now.

As far as commenting that one line of code, that was me trying to figure out why my bug was happening in the first place. That toggle was not there originally so I went ahead and removed it. 

Resizing the cursor size should be working as expected now and visibility of the smallest cursor has been improved.

ab28dev commented on LiveDraw/MainWindow.xaml.cs:
```diff
@@ -872,7 +872,6 @@ private void LineMode(bool l)
         {
             LineButton.IsActived = l;
             _lineMode = l;
-            //EraseByPoint_Flag = (int)erase_mode.NONE;
```
`EraseByPoint_Flag = (int)erase_mode.NONE;`

ab28dev commented:
Erase by point size is now visible while changing size of eraser and doesnot interfere with pen size.
I have reviewed everything and added a commit which solves some issues in toggle views of various buttons.
